README.md

```diff
@@ -17,64 +17,67 @@
 - [Contributing](#contributing)
 - [License](#license)
 
-# Deployments
```

Comment:
existing proxies are managed in hardhat config

Reply:
i thought it would be annoying to have to update these in two places... i made a ref to the `proxies` entry in hardhat config in this file. we can def add back if you guys still want it